File: myblog-server/src/test/java/com/site/blog/BlogAppTests.java
package com.site.blog;

import com.site.blog.model.entity.AdminUser;
import com.site.blog.service.AdminUserService;
import org.junit.jupiter.api.Test;
import org.junit.jupiter.api.extension.ExtendWith;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.test.context.junit.jupiter.SpringExtension;

import javax.annotation.Resource;
import java.io.File;

@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
@ExtendWith({SpringExtension.class})

public class BlogAppTests {
    @Resource
    AdminUserService adminUserService;

    @Test
    public void contextLoads() {
        AdminUser u = adminUserService.getAdminUserById(4);
        System.out.println(u);
    }

    @Test
    public String getUserById() {
        AdminUser u = adminUserService.getAdminUserById(4);
        System.out.println(u);
        return "aaa";
    }


}

File: myblog-kt/src/test/kotlin/com/site/blog/BlogAppTests.kt
package com.site.blog

import com.baomidou.mybatisplus.extension.kotlin.KtQueryWrapper
import com.site.blog.mapper.BlogConfigMapper
import com.site.blog.model.entity.BlogConfig
import org.junit.jupiter.api.Test
import org.springframework.boot.test.context.SpringBootTest
import javax.annotation.Resource


@SpringBootTest

class BlogAppTests() {

    @Resource
    lateinit var blogConfigMapper: BlogConfigMapper

    @Test
    fun getBlogConf() {
        val flag: Boolean = blogConfigMapper.exists(KtQueryWrapper (BlogConfig()).eq(BlogConfig::configCode, "init"))
        println("是否是第一次? ${!flag}")
    }

}
